docs/core/compatibility/globalization/5.0/listseparator-value-change.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Breaking change: TextInfo.ListSeparator value change"
+description: Learn about the .NET 5.0 breaking change in where the default value of TextInfo.ListSeparator was changed between versions 5.0 and 5.0.1.
+ms.date: 12/10/2020
+---
+# TextInfo.ListSeparator values changed
+
+The default <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> values for different cultures have been reverted to their pre-.NET 5.0 values.
+
+## Change description
+
+In .NET 5.0.0, the default <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> values for different cultures changed. On Windows, the values changed as part of the [Globalization APIs use ICU libraries on Windows](icu-globalization-api.md) change. On Linux and macOS, instead of using decimal separators, the values for <xref:System.Globalization.TextInfo.ListSeparator?displayProperty=nameWithType> were also obtained from International Components for Unicode (ICU).
```

Comment:
I think we need to rewrite this part. here is some clarification:

When using .NET 5.0 on Windows and not switching to NLS behavior, the ListSeparator value will be same as the Decimal separator value which we read from ICU. If using any .NET Core version up to and including 5.0 and running on Linux/Mac, the behavior will be same as when running with 5.0 on Windows.